docs/src/components/DocLayout.tsx

```diff
@@ -65,16 +76,39 @@ const ContentContainer = styled(Box)`
   margin-top: 0;
 `;
 
+const TocWrapper = styled.div`
+  ${mediaQueries.tablet(css`
+    order: 2;
+  `)}
+
+  padding-left: ${({ theme }) => theme.orbit.spaceSmall};
+  > * {
+    position: sticky;
```

Comment:
I can't figure out why this doesn't work, I'll need an explanation once again how it's connected to `overflow: hidden`.

Reply:
It's solved with #2856